### DISCUSSION TITLE
How do I set the global wait time for steps in the specs case

### DISCUSSION THREAD

**Discussion by curry-cb** [getgauge/gauge#2296] [General]
gauge Where can I set the delay for each step,like this
![image](https://user-images.githubusercontent.com/75304638/195502850-a1a660f4-b533-41ec-a353-d15c07a36a2c.png)

**Comment by lirany1** [comment]
you can do it like this:
in your spec:
```
* Run test 1
* Sleep for "80000"
* Run test 2

```
then in the implementation (Java/Kotlin code) 

```
	@Step("Sleep for <time>")
	fun sleep(milliseconds: Long) {
		Thread.sleep(milliseconds)
	}
```

**Reply by curry-cb** [reply]
I would like to have this waiting time at each step, because it would be too cumbersome to add
